### ISSUE TITLE
Fix security issues

### ISSUE THREAD

**Issue by alekseynemiro** [alekseynemiro/multi-duper-stopwatch#88]
`npm audit fix`

Please check: https://github.com/alekseynemiro/multi-duper-stopwatch/security

**Comment by alekseynemiro** [comment]
```
npm audit
# npm audit report

fast-xml-parser  <4.2.4
Severity: high
fast-xml-parser vulnerable to Regex Injection via Doctype Entities - https://github.com/advisories/GHSA-6w63-h3fj-q4vw
fix available via `npm audit fix`
node_modules/fast-xml-parser

semver  <=5.7.1 || 6.0.0 - 6.3.0 || 7.0.0 - 7.5.1
Severity: moderate
semver vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-c2qf-rxjj-qqgw
semver vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-c2qf-rxjj-qqgw
semver vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-c2qf-rxjj-qqgw
fix available via `npm audit fix`
node_modules/@babel/register/node_modules/semver
node_modules/@mapbox/node-pre-gyp/node_modules/semver
node_modules/@npmcli/fs/node_modules/semver
node_modules/@opentelemetry/instrumentation/node_modules/semver
node_modules/@react-native-community/cli-clean/node_modules/semver
node_modules/@react-native-community/cli-doctor/node_modules/cross-spawn/node_modules/semver
node_modules/@react-native-community/cli-platform-android/node_modules/semver
node_modules/@react-native-community/cli-platform-ios/node_modules/semver
node_modules/@react-native-community/cli-plugin-metro/node_modules/semver
node_modules/@react-native-community/cli/node_modules/cross-spawn/node_modules/semver
node_modules/@react-native-windows/cli/node_modules/semver
node_modules/@typescript-eslint/eslint-plugin/node_modules/semver
node_modules/@typescript-eslint/typescript-estree/node_modules/semver
node_modules/@typescript-eslint/utils/node_modules/semver
node_modules/async-listener/node_modules/semver
node_modules/cls-hooked/node_modules/semver
node_modules/diagnostic-channel/node_modules/semver
node_modules/find-cache-dir/node_modules/semver
node_modules/jest-snapshot/node_modules/semver
node_modules/node-abi/node_modules/semver
node_modules/node-gyp/node_modules/semver
node_modules/semver
node_modules/ts-jest/node_modules/semver
node_modules/username/node_modules/semver

word-wrap  <1.2.4
Severity: moderate
word-wrap vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-j8xg-fqg3-53r7
fix available via `npm audit fix`
node_modules/word-wrap

3 vulnerabilities (2 moderate, 1 high)
```

**Comment by alekseynemiro** [comment]
```
npm audit fix

removed 59 packages, changed 26 packages, and audited 1655 packages in 5s

143 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by alekseynemiro** [comment]
Fixed in v1.2.
